CMakeLists.txt

```diff
@@ -142,11 +169,18 @@ set(SYCLCC_CONFIG_FILE "{
   \"default-gpu-arch\"  : \"${DEFAULT_GPU_ARCH}\",
   \"default-cpu-cxx\"   : \"${CMAKE_CXX_COMPILER}\",
   \"default-rocm-path\" : \"${ROCM_PATH}\",
+  \"default-boost-path\" : \"${BOOST_PATH}\",
```

Comment:
Is this still actually used? It seems that `syclcc` doesn't have this option?

Reply:
No it is not, I overlooked this one